Decred Wallet/Features/Receive/ReceiveViewController.swift

```diff
@@ -65,11 +62,48 @@ class ReceiveViewController: UIViewController,UIDocumentInteractionControllerDel
         super.viewWillDisappear(animated)
         self.navigationItem.leftBarButtonItem = nil
     }
+    
+    @objc func accountSelectionDropdownBtnClick() {
+        self.accountSelectionDropdownBtn.isSelected = !self.accountSelectionDropdownBtn.isSelected
+        
+        if self.receiveAccountTableView == nil {
```

Comment:
Make `receiveAccountTableView` a lazy variable. After that you can delete `createReceiveAccountTableView` function. That way, the variable will only be initialized in line 73 calling the showView function.
```
    private lazy var receiveAccountTableView: AccountTableView = {
        let accountTableView = AccountTableView.init(frame: CGRect(x: 0, y: 0, width: self.view.frame.width, height: self.view.frame.height))
    UIApplication.shared.keyWindow?.addSubview(accountTableView)
    
        accountTableView.selectedAccount = {
            [weak self] (account:DcrlibwalletAccount,walletName:String) in
            
            guard let this = self else { return }
            
            this.myacc = account
            
            this.selectedAccountNameLbl.text = this.myacc?.name
            this.selectedAccountWalletNameLbl.text = walletName
            
            let total = "\(this.myacc?.balance?.total ?? 0)"
            this.selectedAccountTotalBalanceLbl.attributedText = Utils.getAttributedString(str: "\(total)", siz: 14.0, TexthexColor: UIColor.init(hex: "0A1440"))
                            
            this.getAddress(accountNumber: (this.myacc!.number))
        }
        accountTableView.hide = { [weak self] () in
            guard let this = self else { return }
            this.accountSelectionDropdownBtn.isSelected = !this.accountSelectionDropdownBtn.isSelected
        }
        return accountTableView
    }()```